versions.tf

```
@@ -22,7 +22,7 @@ terraform {
     }
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = "4.75.0"
+      version = "4.81.0"
```

I recommend adding the pessimistic constraint operator so this isn't a breaking change.

```suggestion
      version = "~> 4.75"
```